src/ImageSharp/Formats/Tiff/PhotometricInterpretation/WhiteIsZero1TiffColor{TPixel}.cs

```diff
@@ -24,11 +24,11 @@ public override void Decode(ReadOnlySpan<byte> data, Buffer2D<TPixel> pixels, in
         colorBlack.FromRgba32(Color.Black);
         colorWhite.FromRgba32(Color.White);
         ref byte dataRef = ref MemoryMarshal.GetReference(data);
-        for (nint y = top; y < top + height; y++)
+        for (nint y = top; y < (nint)(uint)(top + height); y++)
```

Comment:
In some places we haven't dont the second explicit cast from uint
e.g. the loops are written as 

```c#
for (nint y = top; y < (uint)(top + height); y++)
```

I think we should strive to be super consistent here.

Reply:
Having another thought on this, I think using `nuint` is the best option.
* the cast to `uint` is enough, so no need for `(nint)(uint)`
* the compiler errors for `for (nuint i = 0; i < span.Length; ++i)` whilst there's no error if `nint i = 0` is used -- so it's impossible to miss one optimization
* for 32-bit optimized code is emitted too

Hence I'll update the usages to `nuint`.